providers/gce/gce.go

```diff
@@ -393,6 +404,10 @@ func generateCloudConfig(configFile *ConfigFile) (cloudConfig *CloudConfig, err
 		cloudConfig.SecondaryRangeName = configFile.Global.SecondaryRangeName
 	}
 
+	if configFile != nil {
+		cloudConfig.StackType = configFile.Global.StackType
```

Comment:
What would be the default value for this `StackType` field? Is empty the same as `IPV4`?

Reply:
We should treat empty the same as IPV4